src/appleseed-max-impl/utilities.cpp

```diff
@@ -171,18 +171,23 @@ std::string insert_texture_and_instance(
     }
 
     const std::string texture_name = wide_to_utf8(bitmap_tex->GetName());
-    base_group.textures().insert(
-        asr::DiskTexture2dFactory::static_create(
+    if (base_group.textures().get_by_name(texture_name.c_str()) == nullptr)
```

Comment:
As you hinted in your PR's description, doesn't this create a bug when two 3ds Max maps have the same name? If so, we'll need to track maps properly, I'm afraid.

In any case we can keep these changes for now.